README.md

```diff
@@ -1,158 +1,111 @@
-
-1. The admin creates the Golden PVC in the "golden" namespace.  This PVC should either reference a desired Storage Class or fall to the cluster default.  These PVCs, annotated per below, signal the controller to launch the ephemeral importer pod.  
-
-1. When a PVC is created, the dynamic provisioner referenced by the Storage Class will create a Persistent Volume representing the backing storage volume.
+This project is designed with Kubevirt in mind and provides a declarative method of importing VM images into a Kuberenetes cluster. Through this behavior, cluster administrators can build an abstract registry of template images (referred to as "Golden Images" for their role as templates for image clones).
 
-1. (Parallel to 4) The dynamic provisioner creates the backing storage volume.
+For an in depth look at the system and workflow, see the [Design](/doc/design.md#design) documentation.
 
-    >NOTE: for VM images there is a one-to-one mapping of a volume to a single image. Thus, each VM image has one PVC and one PV defining it.
+### Data Format
 
-1. The Data Import Pod, created by the controller, binds the Secret and mounts the backing storage volume via the Persistent Volume Claim.
+The importer is capable of performing certain functions that streamline its use with Kubevirt.  It will unpackage **gzip** and **tar** archived files automatically, as well as convert **qcow2** images into raw image files.
 
-1. The Data Importer Pod streams the file from the remote data store to the mounted backing storage volume. When the copy completes the importer pod terminates. The destination file name is always _disk.img_ but, since there is one volume per image file, the parent directory will (and must) differ.
+Expected file formats are:
 
-### Components
+- .tar
+- .gz
+- .img
+- .iso
+- .qcow2
 
-**Data Import Controller:** Long-lived Controller pod in "golden" namespace.
-The controller scans for "golden" PVCs in the same namespace looking for specific
-annotations:
-- kubevirt.io/storage.import.endpoint:  Defined by the admin: the full endpoint URI for the source file/image
-- kubevirt.io/storage.import.secretName: Defined by the admin: the name of the existing Secret containing the credential to access the endpoint.
-- kubevirt.io/storage.import.status: Added by the controller: the current status of the PVC with respect to the import/copy process. Values include:  ”In process”, “Success”, “ Failed”
+## Deploying CDI
 
-On detecting a new PVC with the endpoint annotation (and lacking the status annotation), the controller creates the Data Importer pod "golden" namespace. The controller performs clean up operations after the data import process ends.
+### Assumptions
+- A running Kubernetes cluster
+- A storage class and provisioner.
+- An HTTP or S3 file server hosting VM images
+- A namespace acting as the image registry (`default` is fine for tire kicking)
```

Comment:
maybe mention that the manifest examples cannot be used as-is. They need to be copied locally and edited (or at least reviewed) prior to use.

Reply:
This is covered in depth around line 68